packages/ai-core/README.md

```diff
@@ -0,0 +1,166 @@
+# @sap-ai-sdk/ai-core
+
+This package provides tools to manage your scenarios and workflows in SAP AI Core.
+
+Streamline data preprocessing and model training pipelines
+Execute batch inference jobs
+Deploy inference endpoints for your trained models
+Register custom Docker registries, sync AI content from your own git repositories, and register your own object storage for training data and model artifacts
+
+### Installation
+
+```
+$ npm install @sap-ai-sdk/ai-core
+```
+
+## Pre-requisites
```

Comment:
[q] Can we add this to the root README and link here instead to make it easier to maintain?